### ISSUE TITLE
All resolve aliases should be self-contained in the addons-registry

### ISSUE THREAD

**Issue by tiberiuichim** [plone/volto#5309]
https://github.com/plone/volto/blob/8caa10f07fbc2fcc97a3bfb95af80e4c39cb763f/razzle.config.js#L225-L245

For example, `@plone/volto-original` is not here https://github.com/plone/volto/blob/8caa10f07fbc2fcc97a3bfb95af80e4c39cb763f/packages/generator-volto/generators/app/templates/.eslintrc.js#L37-L44

**Comment by davisagli** [comment]
@tiberiuichim I'm not sure I understand the issue. Are you saying that the eslint config should be deriving more of its `map` configuration from the registry?

**Comment by tiberiuichim** [comment]
@davisagli Yes, there's too much duplicated code with miniscule variations. Using the addons-registry (or the future `@plone/registry`) package would ensure that we get consistency and less bugs.

**Comment by DhanushShettyH** [comment]
Can I do this.
